packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx

```diff
@@ -108,7 +109,7 @@ export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProp
     const isCustom = strategy === 'custom';
     const allChecked = connectedConfirmed && termsConfirmed;
     const allAnonymized = notAnonymized === '0';
-    const isDisabled = !allChecked || allAnonymized;
+    const isDisabled = !allChecked || allAnonymized || isLocked(false);
     const buttonTooltipMessage = allAnonymized
```

Comment:
Would be nice to adapt tooltip message for the locked state (show no tooltip message or something like "In progress"), but the state should be there only for a very short time, right? So it's probably just a nitpick.

Reply:
this reason for disabling is relevant only for a split of second (until modal appears). If I was to touch it here, I would suggest doing something like this [e97b9ed](https://github.com/trezor/trezor-suite/pull/7196/commits/e97b9ed1b8e201064ce5f7134a7f259e7c80aebd)